src/parser/grammar.y

```diff
@@ -633,7 +635,7 @@ relation(A) ::= NE. { A = NE; }
 // raw value tokens - int / string / float
 value(A) ::= INTEGER(B). {  A = SI_LongVal(B.longval); }
 value(A) ::= DASH INTEGER(B). {  A = SI_LongVal(-B.longval); }
-value(A) ::= STRING(B). {  A = SI_ConstStringVal(B.strval); }
+value(A) ::= STRING(B). {  A = SI_ConstStringVal(B.strval); } // TODO this is probably a leak
```

Comment:
Seems like it, strval is heap allocated by the lexer, but won't be free because it is marked here as const.